.devcontainer/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.241.1/containers/rust/.devcontainer/base.Dockerfile
+
+# [Choice] Debian OS version (use bullseye on local arm64/Apple Silicon): buster, bullseye
+ARG VARIANT="buster"
```

Comment:
What about using bullseye everywhere in order to don't bother with the different architectures? It's stable since a year.

Reply:
In the `devcontainer.json`, it actually uses `bullseye` on line 10. The `buster` value here is just the default value generated by VS Code.

Reply:
I see. What about usinb bullseye by default to simplify the configuration?

Reply:
Sounds good, done!